From 7f2b5331b2c57073bfa618a6141c9950e8ff4771 Mon Sep 17 00:00:00 2001
From: Ciprian Marian Costea <ciprianmarian.costea@nxp.com>
Date: Wed, 18 Aug 2021 11:20:18 +0300
Subject: [PATCH 01/19] s32gen1: qspi: Print read speed info on debug level

commit 7f2b5331b2c57073bfa618a6141c9950e8ff4771 from
https://source.codeaurora.org/external/autobsps32/linux

Print information regarding read speed only on debug level.

Upstream-Status: Pending

Issue: ALB-7554
Signed-off-by: Ciprian Marian Costea <ciprianmarian.costea@nxp.com>
Signed-off-by: Zhantao Tang <zhantao.tang@windriver.com>
---
 drivers/spi/s32gen1-qspi.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/drivers/spi/s32gen1-qspi.c b/drivers/spi/s32gen1-qspi.c
index ba3813138853..b079c6c4350d 100644
--- a/drivers/spi/s32gen1-qspi.c
+++ b/drivers/spi/s32gen1-qspi.c
@@ -1046,8 +1046,8 @@ int qspi_read_mem(struct fsl_qspi *q,
 	if (us_passed > 0) {
 		mb_int = div_u64_rem(op->data.nbytes, us_passed, &rem);
 		mb_frac = div64_u64(rem * 1000, us_passed);
-		dev_info(q->dev, "%u bytes read in %u us (%llu.%llu MB/s)\n",
-				op->data.nbytes, us_passed, mb_int, mb_frac);
+		dev_dbg(q->dev, "%u bytes read in %u us (%llu.%llu MB/s)\n",
+			op->data.nbytes, us_passed, mb_int, mb_frac);
 	}
 
 	qspi_writel(q, mcr_reg, base + QUADSPI_MCR);
-- 
2.17.1

